help/_posts/1970-01-01-homebrew.md

```diff
@@ -83,7 +83,7 @@ git -C "$(brew --repo homebrew/command-not-found)" remote set-url origin https:/
 
 # 或使用下面的几行命令自动设置
 BREW_TAPS="$(brew tap)"
-for tap in core cask{,-fonts,-drivers,-versions}; do
+for tap in core cask{,-fonts,-drivers,-versions,} command-not-found; do
```

Comment:
`cask{,-fonts,-drivers,-versions,}` would be expanded to `cask cask-fonts cask-drivers cask-versions cask`.

Use `cask{,-fonts,-drivers,-versions}`

Reply:
;) sorry for the typo, fixed!